### ISSUE TITLE
T9 layout everywhere

### ISSUE THREAD

**Issue by maverick74** [dslul/openboard#379]
Would be great to able to use the T9 layout (with text prediction and all the other things available for the other layouts) in any regular writing process (texting SMS's, for example...)

![imagem](https://user-images.githubusercontent.com/14912461/120205131-8241ed00-c221-11eb-860e-0b112e7eb329.png)

this is already available in the dialup screens , but we cannot choose it from the keyboard schemes to use it everywhere else...

**Comment by Xcl4m4t10n** [comment]
+1 we really need T9 😄

**Comment by maverick74** [comment]
@dslul @MajeurAndroid can someone, please, give some love to this (huge) request?

I REALLY want to switch TouchPal for something open......

I understand this is a huge request....... :(
but maybe you can help...

**Comment by fransfelix** [comment]
+1 for T9 anywhere!

**Comment by GiorgosXou** [comment]
Please make this possible.

**Comment by GiorgosXou** [comment]
In the form of something like this?
![Uploading Screenshot_2023-04-26-23-59-00-577_org.bromite.bromite.jpg…]()

**Comment by GiorgosXou** [comment]
In the form of something like this?
![Screenshot_2023-04-26-23-59-00-577_org bromite bromite](https://user-images.githubusercontent.com/11987271/234701143-061cca00-1a06-420e-97f8-4692d36e73cf.jpg)

**Comment by GiorgosXou** [comment]
Something like this 
![Screenshot_2023-04-26-23-59-00-577_org bromite bromite_1](https://user-images.githubusercontent.com/11987271/234701836-52c3e33d-fb11-442b-a2d2-5bf5a7ababd8.jpg)

**Comment by ghost** [comment]
I just created an account while searching for a keyboard with that layout. Would be a great feature on openboard!

**Comment by maverick74** [comment]
yeah... there is no OpenSource T9 keyboard with predictive text anywhere!!!

I've searched all over. The only options are closed source :(

**Comment by GiorgosXou** [comment]
At the momment the only option is TouchPal with as many as possibly disabled permisions and especially disabled network activity

**Comment by maverick74** [comment]
There are other (closed source) alternatives but TouchPal is the best, no doubt!

I just don't trust it any of them...

There is a lot of people asking for this (in other similar open-source projects), so there's "market" for it!!! 
It just seems there is a lot of work to do to deploy this specific feature and so it gets postponed or rejected.

There was even a case where a dev that argued it would turn the app too heavy, bloated and that it was something very hard to do... to which another user - gracefully - responded something like "_and yet they did it on old dumb nokia 3310_"

(i've been asking for this for years now...)

**Comment by gylas** [comment]
+1!
